airflow/utils/pydantic.py

```diff
@@ -24,19 +24,15 @@
 
 from __future__ import annotations
 
+from importlib.metadata import PackageNotFoundError, version
 
-def is_pydantic_2_installed() -> bool:
-    import sys
+from pkg_resources import parse_version
```

Comment:
Why do you use `from pkg_resources import parse_version` here and `from packaging.version import parse as parse_version`? Is that an IDE auto-suggestion?

Reply:
Ah no - this is exactly why I do not like `import as` and illustrates why you should not use it usually.

When you copy-paste code, you might accidentally miss that there is also an `import as` to be copied.  Unless those are well known `as` (like `np` or `pd`) I have personally a strong preference of not using `import as` - mostly because they require you to always look at two - usually very far away from each other - places in the python file to understand what's going on `really'. I replaced it all with module imports and prefixing the modules where the generic `version` or `parse` are used. That's way less ambiguous.